testsuite/MDAnalysisTests/formats/test_libdcd.py

```diff
@@ -322,14 +323,18 @@ def write_dcd(in_name, out_name, remarks='testing', header=None):
                     and sys.maxsize <= 2**32) or
                     platform.machine() == 'aarch64',
                    reason="occasional fail on 32-bit windows and ARM")
+# occasionally fails due to unreliable test timings
+@hypothesis.settings(deadline=None)  # see Issue 3096
```

Comment:
Does this remove the need for the xfail above?

Reply:
Hmm let's give it a try

Reply:
I guess we're still not testing on ARM64, so it'd just be the 32bit Windows stuff.

Reply:
@IAlibay no idea why but it makes the Travis Py2.7 test fail again, so the xfail can be left for `develop` instead....